File: free_tasks/search_insert_position.py
class Solution:
    nums = [1,3,6]
    target = 7
    def searchInsert(nums, target: int) -> int:
        for i in range(len(nums)): 
            if target == nums[i]: 
                return i
            else: 
                nums.append(target)
                nums.sort()
                for i in range(len(nums)):
                    if target == nums[i]: 
                        return i
    print(searchInsert(nums, target))

File: free_tasks/ft_js/search_insert_position.js
class Solution {
    static searchInsert(nums, target) {
        for (let i = 0; i < nums.length; i++) {
            if (target === nums[i]) {
                return i;
            } else {
                nums.push(target);
                nums.sort((a, b) => a - b);
                for (let i = 0; i < nums.length; i++) {
                    if (target === nums[i]) {
                        return i;
                    }
                }
            }
        }
    }
}

const nums = [1, 3, 6];
const target = 7;
console.log(Solution.searchInsert(nums, target));
